RELEASE-NOTES.md

```diff
@@ -1,5 +1,29 @@
 # Prefect Release Notes
 
+## Release 2.10.20
+
+### Resolving UI form input issues
+
+This off-cycle release resolves bugs preventing UI form inputs from being rendered and parsed correctly, including:
+- Dates & times — https://github.com/PrefectHQ/prefect-ui-library/pull/1554
+- List values — https://github.com/PrefectHQ/prefect-ui-library/pull/1556
+- JSON fields — https://github.com/PrefectHQ/prefect-ui-library/pull/1557
+
+### Prefect no longer supports Python 3.7
+
+Python 3.7 reached end-of-life on 27 Jun 2023. Consistent with our warning, this release drops Python 3.7 support. Prefect now requires Puython 3.8 or later.
+
+### Enhancements
+- Add UUID validation for webhook CLI commands to raise errors earlier and more clearly — https://github.com/PrefectHQ/prefect/pull/10005
+- Clarify Dockerfile rename language in `prefect deploy` CLI prompt — https://github.com/PrefectHQ/prefect/pull/10124
```

Comment:
```suggestion
- Clarify Dockerfile rename prompt in `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/10124
```
just a suggestion.. not totally sure if it's improved

Reply:
Yours is clearer